website/content/docs/agent-and-proxy/agent/versions.mdx

```diff
@@ -1,11 +1,12 @@
 ---
 layout: docs
-page_title: Vault Agent Version Compatibility
-description: |-
-  Guidelines for running different versions of Agent and Vault server
+page_title: Risks of using inconsistent versions of Agent and Vault
```

Comment:
We might want to include Vault Proxy in scope for this SEO update, as there are a lot of similar descriptions, so if it affects Agent, it will affect Proxy too

Reply:
The entire set of Vault docs is in scope for the SEO updates :)